package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "goatcounter",
+  "version": "4.0.0",
```

Comment:
This intends to match [these versions](https://www.goatcounter.com/help/countjs-versions) and could be bumped accordingly in the future.